src/coreclr/System.Private.CoreLib/src/System/Reflection/MethodBaseInvoker.CoreCLR.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection.Emit;
+
+namespace System.Reflection
+{
+    internal partial class MethodBaseInvoker
+    {
+        private readonly Signature? _signature;
```

Comment:
It doesn't look like this can ever be null based on use cases below, both use `!`. Instead I would make this non-null as it is cleaer.

Reply:
Same for `MethodInvoker` below.

Reply:
It can't be null, but due to constructor chaining the analyzer doesn't detect it. It doesn't seem right to add a suppression for that in the main\base constructor since that could affect other cases.